src/helpers.ts

```diff
@@ -94,7 +94,7 @@ export const isModalOpen = (name: string): boolean => {
 };
 
 export const openModal = async ({ name, params, ...props }: openModalProps) => {
-  const urlParams = new URLSearchParams(cleanSearchParams());
+  const urlParams = cleanSearchParams();
```

Comment:
`cleanSearchParams()` already returns a `URLSearchParams` object, we don't need to instantiate a new one.